store/auth.go

```diff
@@ -31,6 +31,8 @@ var (
 	// MyAppsPackageAccessAPI points to MyApps endpoint to get a package access macaroon
 	MyAppsPackageAccessAPI = myappsAPIBase + "api/2.0/acl/package_access/"
 	ubuntuoneAPIBase       = authURL()
+	// UbuntuoneLocation is the Ubuntuone location as defined in the store macaroon
+	UbuntuoneLocation = authLocation()
```

Comment:
Why so much indirection for this variable?  Isn't that a hardcoded caveat ID that we cannot touch?  If so, can we please have this in a const named `loginCaveatLocation` right next to that function?